### PR TITLE
Format files

### PR DESCRIPTION
**Pull request by hatamiarash7**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
```
Comment (pentamassiv) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: How come you want to change the ISSUE_TEMPLATE files? They are exactly how Github creates them when you use the UI.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-title: ''
```
Comment (pentamassiv) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: How come you want to change the ISSUE_TEMPLATE files? They are exactly how Github creates them when you use the UI.

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -1,10 +1,9 @@
 ---
 name: Question
 about: Ask your Question here
-title: ''
+title: ""
```
Comment (pentamassiv) [inline .github/ISSUE_TEMPLATE/question.md:4]: How come you want to change the ISSUE_TEMPLATE files? They are exactly how Github creates them when you use the UI.

File: src/dsl.rs [hunk 4]
```diff
@@ -41,7 +41,7 @@ impl fmt::Display for ParseError {
     }
 }
 
-/// Evaluate the DSL. This tokenizes the input and presses the keys.
```
Comment (pentamassiv) [inline src/dsl.rs:44]: I think "tokenizes" is correct in this instance?

**Comment by pentamassiv** [comment]
Thank you for the time you put into this PR. Unfortunately your PR was not reviewed in time and now has some conflicts. There are also a few changes that I believe are not needed. I took the pieces that I think are worth merging, created a new commit with them and mentioned you in the commit message (https://github.com/enigo-rs/enigo/pull/133/commits/f467471ac7280d013212fb66236e8a8d55ad847f). The PR containing the commit will be merged once we merged some of the other PRs to avoid conflicts with them.

I hope that is okay with you?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (hatamiarash7) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: 'markdownlint` standards made all the Markdown changes in this PR. Also, other linters/formatters tools use double quotes too.

If we behave similarly to the standards used by the tools, users do not need to be bothered when participating and constantly delete these tiny changes in their PRs. It should be considered that many users use formatting tools, and what Github makes is not necessarily the same as what users use.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (hatamiarash7) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: 'markdownlint` standards made all the Markdown changes in this PR. Also, other linters/formatters tools use double quotes too.

If we behave similarly to the standards used by the tools, users do not need to be bothered when participating and constantly delete these tiny changes in their PRs. It should be considered that many users use formatting tools, and what Github makes is not necessarily the same as what users use.

File: .github/ISSUE_TEMPLATE/question.md (hunk 3 above)
Comment (hatamiarash7) [reply .github/ISSUE_TEMPLATE/question.md:4]: 'markdownlint` standards made all the Markdown changes in this PR. Also, other linters/formatters tools use double quotes too.

If we behave similarly to the standards used by the tools, users do not need to be bothered when participating and constantly delete these tiny changes in their PRs. It should be considered that many users use formatting tools, and what Github makes is not necessarily the same as what users use.

File: src/dsl.rs (hunk 4 above)
Comment (hatamiarash7) [reply src/dsl.rs]: Yes, I was wrong

**Comment by pentamassiv** [comment]
Okay, then I approve merging this PR once we handled some of the other PRs. I have removed the commit https://github.com/enigo-rs/enigo/commit/f467471ac7280d013212fb66236e8a8d55ad847f from my PR.

This PR will have to be rebased though. Once we are ready to merge this PR, I will ask you to rebase it so you don't have to do it multiple times

**Comment by pythoneer** [comment]
I agree with most of it but code formatting should be handled exclusively by `rustfmt` and we should also incorporate CI testing for it and fail if the source code is not formatted the right way (rustfmt way). I am ok with doing it that way for now.

**Comment by pentamassiv** [comment]
We have a Workflow that checks formatting [here](https://github.com/enigo-rs/enigo/blob/master/.github/workflows/format.yml). It keeps failing though, because there are many clippy lints. Those lints will be fixed once https://github.com/enigo-rs/enigo/pull/133 is merged

**Pull request merged**